src/relay/backend/contrib/codegen_c/codegen_c.h

```diff
@@ -190,7 +209,12 @@ class CodegenCBase {
    */
   std::string JitImpl(const std::string& ext_func_id, const Array<Var>& args,
                       const std::vector<std::string>& buf_decl,
-                      const std::vector<std::string>& body, const std::vector<Output>& outs) {
+                      const std::vector<std::string>& body, const std::string& const_arr,
```

Comment:
`const_arr` seems a bit confusion. Maybe `const_arr_name` or something like that. (ditto in some other places)